src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -675,7 +678,7 @@ type Set(value: obj) =
         b.Set value model
         true
     | SubModelSelectedItem b ->
-        b.TypedSet(model, ValueOption.ofObj value)
+        b.TypedSet(model, ValueOption.ofNull value)
```

Comment:
From https://github.com/elmish/Elmish.WPF/pull/470#discussion_r963187690:

> Can you explain how this changes the behavior?

-- @TysonMN 

> Sure.
> 
> So ValueOption.ofObj assumes that value is a nullable type, then wraps it into ValueSome/ValueNone. ValueOption.ofNull does the exact same thing for nullable types, but also handles non-nullable types by just always wrapping them into ValueSome. So it increases the domain of the overall function.

-- @marner2

Reply:
See my reply as part of above comment